src/stackTools/stackScroll.js

```diff
@@ -30,7 +30,7 @@ function mouseDownCallback (e, eventData) {
 function mouseWheelCallback (e, eventData) {
   const images = -eventData.direction;
 
-  scroll(eventData.element, images);
+  scroll(eventData.element, images, stackScroll.getConfiguration().scrollLoop);
```

Comment:
Can you set a default value for this configuration setting? And change it just to 'loop'. I guess that's clear enough, since it's already on the scroll tool.

Reply:
done :)